lxd/project/permissions.go

```diff
@@ -1174,3 +1176,31 @@ func FilterUsedBy(r *http.Request, entries []string) []string {
 
 	return usedBy
 }
+
+// CheckClusterTargetRestriction check if user is allowed to use '--target' flag
+func CheckClusterTargetRestriction(tx *db.ClusterTx, r *http.Request, projectName string, targetFlag string) error {
+	if rbac.UserIsAdmin(r) {
```

Comment:
Can you put a comment there stating `// Allow server administrators to move instances around even when restricted (node evacuation, ...)` so we have a record of why we're making this exception.